lambda/jvm/src/main/scala/feral/lambda/IOLambdaPlatform.scala

```diff
@@ -45,10 +47,15 @@ private[lambda] abstract class IOLambdaPlatform[Event, Result]
             .head
             .compile
             .lastOrError
-          context <- IO(Context.fromJava(context))
-          _ <- OptionT(apply(event, context, setup)).foldF(IO.unit) { result =>
-            // TODO can circe write directly to output?
-            IO(output.write(encoder(result).noSpaces.getBytes(StandardCharsets.UTF_8)))
+          context <- IO(Context.fromJava[IO](context))
+          _ <- OptionT(lambda(event, context)).foldF(IO.unit) { result =>
+            IO {
+              val json = result.asJson
+              val bb = Printer.noSpaces.printToByteBuffer(json)
+              val ch = Channels.newChannel(output)
+              ch.write(bb)
```

Comment:
I opened https://github.com/circe/circe/pull/1855 which would let us write directly to the `OutputStream`, but not sure what's going on there rn.

Reply:
Looks like there are new maintainers, but they probably need to get settled in :)